tests/allocate.ispc

```diff
@@ -0,0 +1,19 @@
+// rule: skip on arch=genx32
+// rule: skip on arch=genx64
+
+void check_val(uniform int *uniform alloca_ptr, uniform float RET[], uniform float aFOO[]) {
```

Comment:
I assume you meant to make this function `noinline`

Reply:
Done